service/src/test/java/uk/gov/hmcts/reform/fpl/validators/HasDocumentStatusValidatorTest.java

```diff
@@ -0,0 +1,115 @@
+package uk.gov.hmcts.reform.fpl.validators;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.Document;
+
+import javax.validation.ConstraintValidatorContext;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+@ExtendWith(SpringExtension.class)
+class HasDocumentStatusValidatorTest {
+    HasDocumentStatusValidator validator = new HasDocumentStatusValidator();
+
+    @Mock
```

Comment:
Those mocks are definitely not pretty, I will think about how you can improve and directly use your validator.